plugin/trino-example-plugin-parent/pom.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.trino</groupId>
+        <artifactId>trino-root</artifactId>
+        <version>392-SNAPSHOT</version>
+        <relativePath>../../pom.xml</relativePath>
+    </parent>
+
+    <groupId>com.example</groupId>
```

Comment:
Does this still get installed if you run `mvn install`? If so, it needs to be under the Trino group ID. We don't want to add junk to the local repository.

Reply:
I'm now skipping the `maven-install-plugin` for this module, so it doesn't, but it does create a single file in the local Maven repo and I'm not sure why: `~/.m2/repository/com/example/trino-example-plugin/0.1-SNAPSHOT/resolver-status.properties`